global.css

```diff
@@ -33,4 +34,7 @@ h2{
 
 h3{
```

Comment:
Try to use class selectors as much as possible

Reply:
yes agreed, but this was special case. Thank you for reviewing Raghav